### PR TITLE
Leaf/tables

### PR DESCRIPTION
**Pull request by peppacaiou**
Fixed the messed-up url patterns. This code has been tested and certified working.

### CODE REVIEW

File: mesh/profiles/models.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
 from django.db import models
-
+from mesh.accounts.models import Account
 # Create your models here.
+print(str(Account))
```
Comment (misslame) [inline mesh/profiles/models.py:4]: Why this print here?

File: media/image/Clown_Emoji_Download_iPhone_Emojis.png
Comment (misslame) [inline media/image/Clown_Emoji_Download_iPhone_Emojis.png:1]: We only need one if youre going to keep this here

Comment (misslame) [reply media/image/Clown_Emoji_Download_iPhone_Emojis.png:1]: Honestly tho, you should remove all of this

**Review by misslame — CHANGES_REQUESTED** [review]

**Review by PattyCakesxD — APPROVED** [review]
Same concerns as Bob, lgtm otherwise

**Comment by misslame** [comment]
This code already in dev and targeted to wrong branch. 

Reopen if this was a mistake

**Pull request closed without merging**